corehq/apps/repeaters/models.py

```diff
@@ -334,7 +376,10 @@ def get_url(self, repeat_record):
             # adapted from http://stackoverflow.com/a/2506477/10840
             url_parts = list(urlparse.urlparse(url))
             query = urlparse.parse_qsl(url_parts[4])
-            query.append(("app_id", self.payload_doc(repeat_record).app_id))
+            try:
+                query.append(("app_id", self.payload_doc(repeat_record).app_id))
+            except (XFormNotFound, ResourceNotFound):
+                return None
```

Comment:
Yeah I agree it's or at the very least not obviously correct. Mind if we just make a note of it and address this outside this PR though? My thinking is I want to separate as much as possible organizational changes and behavior changes, so that if any bug does come up, I know it's a bug in my reorganization, as opposed to a consequence of an intentional change. Unless of course I'm misunderstanding and you're saying this change makes things _worse_?